src/hotspot/share/jvmci/jvmciCompilerToVM.cpp

```diff
@@ -549,7 +549,7 @@ C2V_END
 C2V_VMENTRY_NULL(jobject, getImplementor, (JNIEnv* env, jobject, ARGUMENT_PAIR(klass)))
   Klass* klass = UNPACK_PAIR(Klass, klass);
   if (!klass->is_interface()) {
-    THROW_MSG_0(vmSymbols::java_lang_IllegalArgumentException(),
+    THROW_MSG_NULL(vmSymbols::java_lang_IllegalArgumentException(),
         err_msg("Expected interface type, got %s", klass->external_name()));
```

Comment:
Unlike in the other cases, the indentation has not been updated. Looks good otherwise.

Reply:
In other cases I was maintaining pre-existing alignment.  Here it was already formatted differently, and I
thought this was a file that used this indentation style (not that I like it).  But looking around, I'm not
finding other uses of this style, so I've aligned the arguments.